### ISSUE TITLE
[mac-v2] config_lowmem.toml — Adafactor + fused_backward_pass profile

### ISSUE THREAD

**Issue by hiyukoim** [hiyukoim/CoppyLora_webUI#19]
## Goal

Add a second config file, `config_lowmem.toml`, for users on memory-constrained machines (16 GB Macs, or 24 GB Macs with lots of apps open). It mirrors `config.toml` but swaps AdamW for Adafactor with `fused_backward_pass = true`, cutting optimizer memory by ~50%.

## Background

Adafactor + `fused_backward_pass`:
- `fused_backward_pass` is gated to Adafactor in sd-scripts (`library/train_util.py:4843-4846`) — it fuses the optimizer step into the backward pass, eliminating a full copy of gradients in memory.
- Adafactor itself uses ~50% less optimizer state than AdamW (factored second-moment vs full).
- Convergence is slightly slower per step but quality matches AdamW after sufficient steps. This profile is **opt-in** and clearly labelled as a different convergence path; the plan's quality firewall does not apply since the user explicitly chose a different optimizer.

## File contents

Identical to `config.toml` except for these four fields:

```toml
optimizer_type = "Adafactor"
optimizer_args = ["scale_parameter=False", "relative_step=False", "warmup_init=False"]
fused_backward_pass = true
train_batch_size = 1
```

Everything else (resolution, network_dim, lr_scheduler, cache flags, sdpa, etc.) stays identical to `config.toml` so the user can A/B between them by swapping a single command-line flag.

`train_batch_size = 1` is intentional — the whole point of this profile is to stay under 12 GB MPS budget. Batch=1 cuts activation memory roughly in half.

`optimizer_args = [...]` are the Hugging Face / kohya-recommended Adafactor settings for SDXL LoRA — disables internal LR scaling so the explicit `learning_rate = 0.0001` is honoured.

## Wiring into the app

Out of scope for this issue. v2 ships `config_lowmem.toml` as a file the user can swap in manually:

```bash
mv config.toml config_default.toml
cp config_lowmem.toml config.toml
./start.sh
```

A future issue (post-v2 tag) can add a Gradio dropdown to pick the profile or an env var like `COPPYLORA_CONFIG=config_lowmem.toml ./start.sh`.

## Acceptance criteria

- [ ] `config_lowmem.toml` exists at repo root.
- [ ] `python -c "import toml; print(toml.load('config_lowmem.toml'))"` parses without error.
- [ ] Diff against `config.toml` is exactly the four fields above (verifiable with `diff config.toml config_lowmem.toml`).
- [ ] Smoke test (#22) runs this profile and produces a valid `.safetensors`.

## Edge cases

- User on 64 GB Mac uses lowmem profile → still works, just slightly slower convergence than AdamW.
- User on 8 GB Mac (M1 base) tries default → likely OOMs; README (#21) directs them here.

## Tests

- Smoke test #22 covers both default and lowmem profiles.

## Reference

- Adafactor + fused_backward_pass constraint: `sd-scripts/library/train_util.py:4843-4846`
- Adafactor optimizer args (kohya recommended): https://github.com/kohya-ss/sd-scripts#adafactor
